boards/nordic/nrf54lm20dk/doc/index.rst

```diff
@@ -0,0 +1,102 @@
+.. zephyr:board:: nrf54lm20dk
+
+Overview
+********
+
+The nRF54LM20 Development Kit hardware provides support for the Nordic Semiconductor
+nRF54LM20A Arm Cortex-M33 CPU and the following devices:
+
+* :abbr:`SAADC (Successive Approximation Analog to Digital Converter)`
+* CLOCK
+* RRAM
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`TWIM (I2C-compatible two-wire interface master with EasyDMA)`
+* MEMCONF
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`GRTC (Global real-time counter)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
```

Comment:
I think this will need to go away from all the Nordic boards, this is super redundant with the `zephyr:board-supported-hw` table and just eats up tons of real estate.